pkg/controller/operator.go

```diff
@@ -112,6 +113,23 @@ func (s *Support) Run(controller *controllercmd.ControllerContext) error {
 	authorizer := clusterauthorizer.New(configObserver)
 	insightsClient := insightsclient.New(nil, 0, "default", authorizer, configPeriodic)
 
+	// read proxy CA bundle
+	insightsClient.GetTrustedCABundleFunc = func() (*x509.CertPool, error) {
+		caBytes, err := ioutil.ReadFile("/var/run/configmaps/trusted-ca-bundle/ca-bundle.crt")
+		if err != nil {
+			if os.IsNotExist(err) {
+				return nil, nil
+			}
+			return nil, err
+		}
+		if len(caBytes) == 0 {
+			return nil, nil
+		}
+		certs := x509.NewCertPool()
+		certs.AppendCertsFromPEM(caBytes)
```

Comment:
Nitpick: AppendsCertsFromPEM actually returns tuple with `ok` variable, so we should probably check it similarly as other operators do https://github.com/openshift/cluster-authentication-operator/blob/df3160d1ab9f24448c853bfe567c44a2763017cb/pkg/operator2/transport.go#L42-L45